
[ ERROR ]: Caught exception:


File: src/protocols/rosetta_scripts/RosettaScriptsParser.cc:1207
Input rosetta scripts XML file "/home/bunzelh/AIzymes/design/HAB/HAB_refactor_no_cst/4/scripts/RosettaDesign_4.xml" failed to validate against the rosetta scripts schema. Use the option -parser::output_schema <output filename> to output the schema to a file to see all valid options.
Your XML has failed validation.  The error message below will tell you where in your XML file the error occurred.  Here's how to fix it:

1) If the validation fails on something obvious, like an illegal attribute due to a spelling error (perhaps you used scorefnction instead of scorefunction), then you need to fix your XML file.
2) If you haven't run the XML rewriter script and this might be pre-2017 Rosetta XML, run the rewriter script (tools/xsd_xrw/rewrite_rosetta_script.py) on your input XML first.  The attribute values not being in quotes (scorefunction=talaris2014 instead of scorefunction="talaris2014") is a good indicator that this is your problem.
3) If you are a developer and neither 1 nor 2 worked - email the developer's mailing list or try Slack.
4) If you are an academic or commercial user - try the Rosetta Forums https://www.rosettacommons.org/forum


Error messages were:
Error: Premature end of data in tag MOVERS line 66

74:         <EnzRepackMinimize        name="mv_cst_opt"
75:                                   scorefxn_repack="score"
76:                                   scorefxn_minimize="score_final"
77:                                   cst_opt="true"
78:                                   task_operations="tsk_design,tsk_nothing" />
79: 
Error: Premature end of data in tag ROSETTASCRIPTS line 2

74:         <EnzRepackMinimize        name="mv_cst_opt"
75:                                   scorefxn_repack="score"
76:                                   scorefxn_minimize="score_final"
77:                                   cst_opt="true"
78:                                   task_operations="tsk_design,tsk_nothing" />
79: 
------------------------------------------------------------
Warning messages were:
------------------------------------------------------------



AN INTERNAL ERROR HAS OCCURED. PLEASE SEE THE CONTENTS OF ROSETTA_CRASH.log FOR DETAILS.



---------------------------------------------------------------
[ ERROR ]: Error(s) were encountered when running jobs.
1 jobs failed;
Check the output further up for additional error messages.
---------------------------------------------------------------

mv: cannot stat '7vuu_0001.pdb': No such file or directory
Traceback (most recent call last):
  File "/home/bunzelh/AIzymes/design/HAB/HAB_refactor_no_cst/parent/extract_sequence_from_pdb.py", line 24, in <module>
    main(args)
  File "/home/bunzelh/AIzymes/design/HAB/HAB_refactor_no_cst/parent/extract_sequence_from_pdb.py", line 10, in main
    with open(pdb_in, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: '7vuu_RosettaDesign_4.pdb'
